elements/pfe-navigation/demo/index.html

```diff
@@ -112,7 +132,10 @@ <h2>Small Logo</h2>
         </div>
         <ul class="pfe-navigation__menu" id="pfe-navigation__menu">
           <li class="pfe-navigation__menu-item">
-            <a href="#" class="pfe-navigation__menu-link" aria-haspopup="true" aria-expanded="false">
+            <!-- @note: removed aria-haspopup attr bc it should not be used for this type of menu, changed to using a class has-dropdown but this probably needs to be implemented differently -->
+
+            <!-- @todo: move aria and has-dropdown to shadow DOM? -->
```

Comment:
++

Reply:
@wesruv let's work through this together. Not sure of a better way to implement this instead of `.has-dropdown` class and would love to get your thoughts.